Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.py

```diff
@@ -0,0 +1,332 @@
+    """Test the ability to fetch Azure file.
+    Args:
+        client: Client object.
+    Returns:
+        str. ok for success, relevant error string otherwise.
+    """
+    try:
+        download_link = client.get_azure_download_link()
+        client.get_download_file_content_values(download_link)
+
+    except (requests.exceptions.SSLError, requests.ConnectionError, requests.exceptions.HTTPError) as err:
+        demisto.debug(str(err))
+        raise Exception(ERROR_TYPE_TO_MESSAGE[err.__class__] + str(err))
+
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client) -> Tuple[str, Dict, Dict]:
+    """Retrieves indicators from the feed to the war-room.
+
+    Args:
+        client (Client): Client object configured according to instance arguments.
+
+    Returns:
+        Tuple of:
+            str. Information to be printed to war room.
+            Dict. Data to be entered to context.
+            Dict. The raw data of the indicators.
+    """
+    limit = int(demisto.args().get('limit')) if 'limit' in demisto.args() else 10
+
+    indicators, raw_response = fetch_indicators(client, limit)
+
+    human_readable = tableToMarkdown('Indicators from Azure Feed:', indicators,
+                                     headers=['Value', 'Type'], removeNull=True)
+
+    return human_readable, {f'{INTEGRATION_NAME}Feed.Indicator(val.value === obj.value)': indicators}, {
+        'raw_response': raw_response}
+
+
+def fetch_indicators(client: Client, limit: int = -1) -> Tuple[List[Dict], List]:
+    """Fetches indicators from the feed to the indicators tab.
+    Args:
+        client (Client): Client object configured according to instance arguments.
+        limit (int): Maximum number of indicators to return.
+    Returns:
+        Tuple of:
+            str. Information to be printed to war room.
+            Dict. Data to be entered to context.
+            Dict. The raw data of the indicators.
+    """
+    iterator = client.build_iterator()
+    indicators = []
+    raw_response = []
+
+    if limit != -1:
+        iterator = iterator[:limit]
+
+    for indicator in iterator:
+        indicators.append({
+            'value': indicator['value'],
+            'type': indicator['type'],
+            'rawJSON': indicator
+        })
+
+        raw_response.append(indicator)
+
+    return indicators, raw_response
+
+
+def main():
+    """
+    PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    regions_list = argToList(demisto.params().get('regions', ''))
+    if not regions_list:
+        regions_list = ['All']
+
+    services_list = argToList(demisto.params().get('services', ''))
+    if not services_list:
+        services_list = ['All']
+
+    polling_arg = demisto.params().get('polling_timeout', '')
+    polling_timeout = int(polling_arg) if polling_arg.isdigit() else 20
+    insecure = demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy') == 'true'
```

Comment:
I believe `proxy` value in `demisto.params()` is a boolean. Update here accordingly bevakasha.